FirefoxLite_Downloader_Library/src/main/java/com/example/firefoxlitedownloaderlibrary/DownloadFileWorker.java

```diff
@@ -28,23 +28,22 @@ public DownloadFileWorker(@NonNull Context context, @NonNull WorkerParameters
     @Override
     public Result doWork() {
         try {
-            URL u = new URL("https://i.pinimg.com/originals/49/70/17/497017869c892b73b128ff72f2732035.jpg");
-            URLConnection c = u.openConnection();
-            c.connect();
-            int lengthOfFile = c.getContentLength();
-            InputStream in = c.getInputStream();
+            URL url = new URL("http://mattmahoney.net/dc/enwik8.pmd");
```

Comment:
I guess the had-coded url just for test and will not be merged, right?

Reply:
will be removing the hard  coded url and sending a pull request